src/floris/simulation/flow_field.py

```diff
@@ -28,23 +34,23 @@
 
 @attr.s(auto_attribs=True)
 class FlowField(FromDictMixin):
-    wind_shear: float = float_attrib()
     wind_veer: float = float_attrib()
+    wind_shear: float = float_attrib()
+    air_density: float = float_attrib()
+    reference_wind_height: int = int_attrib()
     wind_speeds: NDArrayFloat = attr.ib(converter=attrs_array_converter)
     wind_directions: NDArrayFloat = attr.ib(converter=attrs_array_converter)
-    reference_wind_height: int = int_attrib()
-    air_density: float = float_attrib()
 
     n_wind_speeds: int = attr.ib(init=False)
     n_wind_directions: int = attr.ib(init=False)
 
-    u_initial: NDArrayFloat = attr.ib(init=False)
-    v_initial: NDArrayFloat = attr.ib(init=False)
-    w_initial: NDArrayFloat = attr.ib(init=False)
+    u_initial: NDArrayFloat = attr.ib(default=np.array([], dtype=float))
```

Comment:
Whats the thought for initializing these to empty arrays rather than leaving uninitialized until the `initialize_velocity_field()`?

Reply:
Is there a benefit? I could go either way with this to be completely honest, so I'm happy to take your lead on this one because I can't see any real drawbacks either.